llpc/tool/amdllpc.cpp

```diff
@@ -439,9 +461,12 @@ static Result initCompileInfo(CompileInfo *compileInfo) {
   // We want the default optimization level to be "Default" which is not 0.
   compileInfo->gfxPipelineInfo.options.optimizationLevel = CodeGenOpt::Level::Default;
   compileInfo->compPipelineInfo.options.optimizationLevel = CodeGenOpt::Level::Default;
+  compileInfo->compPipelineInfo.options.forceCsThreadIdSwizzling = ForceCsThreadIdSwizzling;
```

Comment:
Why has setting this option moved inside the #if guard?

Reply:
Good comment. ForceCsThreadIdSwizzling is added in  version, but I have check the legacy code, seem this option is not related to binary version. So I would like to move this part outside #if guard.